libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -332,6 +332,21 @@ export const slackConfig: IConfigCredentials[] = [
   },
 ];
 
+export const whatsappConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.PhoneNumberId,
+    displayName: 'Phone Number ID',
+    type: 'string',
+    required: true,
+  },
```

Comment:
Is this section is the recipient number? or the sender number? In case it's a recipient number, this should be passed dynamically from the `subscriber.phone` just like we do with SMS. Wdyt?

Reply:
This is an ID based on the sender number. You need to supply the `id` of the phone number and the `token`
Then you pass dynamically the sender number and message